### PR TITLE
Fix NPE in case if schema package name is absent

### PR DESCRIPTION
**Pull request by MykytaKostiuk**

### CODE REVIEW

**Review by jemacineiras — APPROVED** [review]

**Comment by jemacineiras** [comment]
It looks good to me, but can you provide more context about?
Thanks

**Review by jemacineiras — CHANGES_REQUESTED** [review]
Version Number need to bump

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <artifactId>kloadgen</artifactId>
 
-  <version>5.3.2</version>
+  <version>5.3.3</version>
```
Comment (jemacineiras) [inline pom.xml:10]: This is a good, one, but have an special POM for Maven Central Deployment. For now we have this 2 files. Hope in the future we can join in one.

**Review by jemacineiras — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (MykytaKostiuk) [reply pom.xml:10]: Updated pom-maven-central.xml as well

**Review by jemacineiras — APPROVED** [review]

**Pull request merged**